test/AbstractObjectTest.cpp

```diff
@@ -50,13 +49,19 @@ void AbstractObjectTest::initEpcDoc()
 		return;
 
 	this->initEpcDocHandler();
+
+	COMMON_NS::AbstractObject* resqmlObject = static_cast<COMMON_NS::AbstractObject*>(this->epcDoc->getDataObjectByUuid(this->uuid));
```

Comment:
From a general point of view (coding rules), usage of "this" is not recommended in fesapi (except if needed)